posthog/special_migrations/setup.py

```diff
@@ -93,7 +92,8 @@ def kickstart_migration_if_possible(migration_name: str, applied_migrations: set
 
     from posthog.special_migrations.runner import run_next_migration
 
-    run_next_migration(migration_name)
+    # start running 30 minutes from now
```

Comment:
nit: provide the reasoning here too - that often after doing some ops we're likely to do others and it's better for migrations to not be interrupted